### DISCUSSION TITLE
Welcome to CoreKeeperMapTool Discussions!

### DISCUSSION THREAD

**Discussion by Ceddini** [Ceddini/CoreKeeperMapTool#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zackyispaul** [comment]
Hi , my map is not the one i downloaded, i can see the right map and the explored part good and very well, and everything is ok, but when i download it , its another map

**Reply by zackyispaul** [reply]
its rotated and the things around the boss island are completely different

**Reply by Ceddini** [reply]
Did you press "Open Full Map Image (Spoilers)" or the "Download Map Image" button? 
The "Open Full Map Image (Spoilers)" only shows an example map as an image.
![grafik](https://user-images.githubusercontent.com/21086465/203934183-dc4229fc-3f13-4711-b8e3-243b19a0ca4a.png)

If you did press the "Download Map Image" button, did you use the development version? The dev version is actively worked on, so there might be some issues with it

**Comment by 6lancmange** [comment]
What is "Crop Rings to Biome"? When I toggle it, I don't see any changes. I'm guessing it should cut out a portion of the ring where the boss shouldn't appear because it's the wrong biome, right? I'm not sure that's happening, although I haven't explored much. The only crop my furthest ring is getting is from the map just cutting off. I wonder, why does it cut off in this way (off-centered)?
![obraz](https://user-images.githubusercontent.com/26997029/204001263-95c0cbd5-5cf9-4354-a42a-c38342459d24.png)

**Reply by Ceddini** [reply]
Crop to biome is a dead button in the normal version. It's been added in the dev version though.

It crops the rings of the pois to the biome they're in. Works in conjunction with the manual rotation thing. 
Has the same "problem" you mentioned in an issue. So if it's enabled but there's not enough data, it'll just show the full ring
